### PR TITLE
add methods for open device in PackedSequence module.

### PR DESCRIPTION
**Pull request by Shan19900305**
1) add is_{custom_device_name}() and {custom_device_name}() for open device register;
2) fix open device failed testcases.

@ezyang  @bdhirsh

### CODE REVIEW

File: test/cpp_extensions/open_registration_extension.cpp [hunk 1]
```diff
@@ -492,7 +492,7 @@ TORCH_LIBRARY_IMPL(aten, PrivateUse1, m) {
   m.impl("sub.Tensor", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
   m.impl("_foreach_add.List", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
   m.impl("index.Tensor", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
-  m.impl("triu.indices", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
+  m.impl("triu_indices", torch::CppFunction::makeFromBoxedFunction<&custom_cpu_fallback>());
```
Comment (ezyang) [inline test/cpp_extensions/open_registration_extension.cpp:495]: whoops

**Review by ezyang — APPROVED** [review]

**Comment by Shan19900305** [comment]
@ezyang Hi, yang. Pls add a label for this pr.

**Pull request closed without merging**